cms/templates/cms/program_page.html

```diff
@@ -5,6 +5,8 @@
 {% load wagtailimages_tags %}
 
 {% block title %}{% trans "MIT Micromasters" %}{% endblock %}
```

Comment:
I'm not sure if I'm reading this correctly, but should we set this to be `{{ page.program.title }}` as well?